eva-release/src/main/java/uk/ac/ebi/eva/release/repositories/ReleaseStatsPerSpeciesRepository.java

```diff
@@ -27,17 +28,21 @@
 
     Iterable<ReleaseStatsPerSpecies> findAllByReleaseVersion(int releaseVersion);
 
-    //All release data excluding rows with only unmapped data
-    Iterable<ReleaseStatsPerSpecies> findByCurrentRsNotAndMultiMappedRsNotAndMergedRsNotAndDeprecatedRsNotAndMergedDeprecatedRsNotAndUnmappedRsGreaterThan(
-            long currentRs, long multiMappedRs, long mergedRs, long deprecatedRs, long mergedDeprecatedRs,
-            long unmappedRs);
-
-    //Data by release version excluding rows with only unmapped data
-    Iterable<ReleaseStatsPerSpecies> findByReleaseVersionAndCurrentRsNotAndMultiMappedRsNotAndMergedRsNotAndDeprecatedRsNotAndMergedDeprecatedRsNotAndUnmappedRsGreaterThan(
-            int releaseVersion, long currentRs, long multiMappedRs, long mergedRs, long deprecatedRs,
-            long mergedDeprecatedRs, long unmappedRs);
-
-    Iterable<ReleaseStatsPerSpecies> findByNewCurrentRsGreaterThan(long currentRs);
-
-    Iterable<ReleaseStatsPerSpecies> findByReleaseVersionAndNewCurrentRsGreaterThan(int releaseVersion, long currentRs);
+    //All released species stats excluding the unmapped only
+    @Query(value = "select * from release_rs_statistics_per_species " +
+            "where not (current_rs = 0 and multi_mapped_rs = 0 and merged_rs = 0 and deprecated_rs = 0 " +
+            "and merged_deprecated_rs = 0 and unmapped_rs > 0)", nativeQuery=true)
+    Iterable<ReleaseStatsPerSpecies> getAllExcludingUnmappedOnly();
+
+    //Species stats by release version excluding the unmapped only
+    @Query(value = "select * from release_rs_statistics_per_species " +
+            "where release_version = ?1 and not (current_rs = 0 and multi_mapped_rs = 0 and merged_rs = 0 " +
+            "and deprecated_rs = 0  and merged_deprecated_rs = 0 and unmapped_rs > 0)", nativeQuery=true)
```

Comment:
Can we refactor the query string for getAllExcludingUnmappedOnly into a variable and just add a release_version condition for this method?